tests/acceptance/features/coreApiShareCreateSpecialToShares2/createShareWithDisabledUser.feature

```diff
@@ -24,5 +24,4 @@ Feature: share resources with a disabled user
     And user "Alice" has been disabled
```

Comment:
Previously this scenarios seems to be failed. And now why it is failing. Please check it and make new issue if possible.